miqa/core/tasks.py

```diff
@@ -130,12 +130,13 @@ def perform_import(import_dict, project_id: Optional[str]):
     Scan.objects.bulk_create(new_scans)
     Frame.objects.bulk_create(new_frames)
 
+    # must use str, not UUID, to get sent to celery task properly
```

Comment:
Is this true? I would have assumed UUIDs would pickle up just fine

Reply:
Without the cast to string below that line, we get the following exception when we run an import: 
```
celery_1    | Traceback (most recent call last):
celery_1    |   File "/usr/local/lib/python3.8/site-packages/kombu/serialization.py", line 39, in _reraise_errors
celery_1    |     yield
celery_1    |   File "/usr/local/lib/python3.8/site-packages/kombu/serialization.py", line 210, in dumps
celery_1    |     payload = encoder(data)
celery_1    |   File "/usr/local/lib/python3.8/site-packages/kombu/utils/json.py", line 68, in dumps
celery_1    |     return _dumps(s, cls=cls or _default_encoder,
celery_1    |   File "/usr/local/lib/python3.8/json/__init__.py", line 234, in dumps
celery_1    |     return cls(
celery_1    |   File "/usr/local/lib/python3.8/json/encoder.py", line 199, in encode
celery_1    |     chunks = self.iterencode(o, _one_shot=True)
celery_1    |   File "/usr/local/lib/python3.8/json/encoder.py", line 257, in iterencode
celery_1    |     return _iterencode(o, 0)
celery_1    | TypeError: keys must be str, int, float, bool or None, not UUID
```

Reply:
Huh, apparently JSON is kombu's default serializer. I think I'll bring this up on the platform channel to see if we want to use pickling instead.